src/i18n/contentProxy.js

```diff
@@ -20,6 +20,10 @@ const contentProxy = (step, prefix) => {
         return isToString || isInspect;
       };
     }
+    if (name === 'keys') {
```

Comment:
Might be better to use the `ownKeys` proxy trap? That way in your tests you can do `Object.keys(content)` to get the list of keys back.

That way a content property called `keys` can still be allowed (I know, why would anyone but you never know).

docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy/handler/ownKeys

Reply:
Woah... yep much better way of doing things... your knowledge of the proxy never fails to impress...

Reply:
🤵 

I spent far too much time looking at the proxy spec last year. It's really hard to make what should be a simple proxy 😞.

Reply:
tried every which way to get this work, but not getting anywhere what i have so far is:
```
const ownKeys = target => {
    const content = target.getResourceBundle(target.language, `${step.name}`);
    return Object.keys(content);
  };

  return { get, ownKeys };
```
but its just returning empty array

Reply:
Exactly what I mean 😞 the proxy spec is really hard to use. That looks to me like it should work, no idea why not.

Maybe try `Reflect.ownKeys(content)`?